spec/factories/category.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :category do
+    name       "Specials"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.